benchmarks/utils.mjs

```diff
@@ -196,21 +193,18 @@ export async function compareSigning(payload, algorithm, privateKey, publicKey)
     [`${algorithm} - fast-jwt (sync)`]: function () {
       fastjwtSign(payload)
     },
-    [`${algorithm} - fast-jwt (async)`]: function (done) {
-      fastjwtSignAsync(payload, done)
-    }
-  })
-
-  Object.assign(tests, {
+    [`${algorithm} - fast-jwt (async)`]: async function () {
+      return fastjwtSignAsync(payload)
+    },
     [`${algorithm} - @node-rs/jsonwebtoken (sync)`]: function () {
       nodeRsSignSync({ data: payload }, privateKey, { algorithm: Algorithm[algorithm.toUpperCase()] })
     },
-    [`${algorithm} - @node-rs/jsonwebtoken (async)`]: function (done) {
-      nodeRsSign({ data: payload }, privateKey, { algorithm: Algorithm[algorithm.toUpperCase()] }).then(() => done())
+    [`${algorithm} - @node-rs/jsonwebtoken (async)`]: async function () {
```

Comment:
strictly speaking, async is not required either unless something's awaited in the body of the function, but ok it's a small detail